sktime/transformers/summarise.py

```diff
@@ -1,12 +1,25 @@
 import numpy as np
 import pandas as pd
+from functools import partial
 from sklearn.utils.validation import check_is_fitted
 
+
 from sktime.transformers.base import BaseTransformer
 from sktime.transformers.segment import RandomIntervalSegmenter
-from sktime.utils.data_container import check_equal_index, tabularize
+from sktime.utils.data_container import check_equal_index, convert_data
 from sktime.utils.validation.supervised import validate_X, check_X_is_univariate
 
+import tsfresh
```

Comment:
In terms of dependency management, we don't want to make tsfresh a strict dependency for all users. This leave us with two options as far as I can see: 
* Move the tsfresh imports into the methods of the transformer, so that people would only need to install tsfresh when they want to use the tsfresh transformers, but everything else in sktime works without it 
* Move the tsfresh transformers into a separate companion repo, I created one [here](https://github.com/sktime/sktime-tsfresh) a while ago, but I now prefer the first option